renderer/src/typings.ts

```diff
@@ -26,6 +26,7 @@ declare global {
         getStationWalletTransactionsHistory: () => Promise<(FILTransaction|FILTransactionProcessing)[]>;
         transferAllFundsToDestinationWallet: () => Promise<void>;
         browseTransactionTracker: (transactionHash: string) => void;
+        openBeryx: () => void;
```

Comment:
I find it surprising that `openBeryx()` is nested under `stationConfig`.

Can we move it one level up, alongside `openReleaseNotes()`?